getting_started.md

```diff
@@ -128,83 +138,112 @@ end
 <br>
 
 
->**Note:** With this pattern, it is trivial to add more complex learning routines that make use of control flow, distributed compute, scheduling optimisation etc. Note that the pattern above is a simple julia *for loop* but it could also be replaced with a *while loop*.
+>**Note:** With this pattern, it is easy to add more complex learning routines that make use of control flow, distributed compute, scheduling optimisations, etc. Note that the pattern above is a simple Julia *for loop* but it could also be replaced with a *while loop*.
 
 <br>
 
-Flux enables you to execute the same process with the [Flux.train!](https://fluxml.ai/Flux.jl/stable/training/training/#Training-1) function. It executes one training step, and you can put the `Flux.train!` function inside a *for loop* to execute more training steps. For more information on training a model in Flux, see [Training](https://fluxml.ai/Flux.jl/stable/training/training/#Training-1). 
+While writing your own loop is powerful, sometimes you just want to do the simple thing without writing too much code. Flux lets you do this with the [Flux.train!](https://fluxml.ai/Flux.jl/stable/training/training/#Training-1) function, which computes a gradient and updates the model for *every* example in a set (running what is often referred to as a *training epoch*). In our case, we could have replaced the entire loop with the following statement:
 
 ```julia
-Flux.train!(loss, params(model), data, opt)
+Flux.train!(loss, params(W_learned, b_learned), train_data, opt)
 ```
 
 <br>
 
-where:
-* **loss** is the loss function that you defined in [Step 3](#step-3-define-a-loss-function).
-* **params(model)** are the trainable parameters of the model. It uses the [`Flux.params` function](https://fluxml.ai/Flux.jl/stable/training/training/#Model-parameters-1) to track the parameters.
-* **data** is a collection of data points. This data must be of the same dimension as the input of the `model` function.
-* **opt** is an optimiser.
-
+For more ways to train a model in Flux, see [Training](https://fluxml.ai/Flux.jl/stable/training/training/#Training-1).
 
 <br>
 
-### Step 7: Run the script
+### Step 7: Examine the Results
+
+The training loop we ran modified `W_learned` and `b_learned` to be closer to the values used to generate the training data (`W_actual` and `b_actual`). We can see how well we did by printing out the difference between the learned and actual matrices.
+
+```julia
+println("The value of W_learned is:")
+display(W_learned)
+print("The largest element-wise difference between W_learned and W_actual is ")
+println(maximum(abs.(W_learned - W_actual))
```

Comment:
Instead of writing about what each value represents in prose, I think a simple `@show` would do the trick:
```suggestion
@show W_learned
@show maximum(abs, W_learned .- W_actual)
```